Sources/Syntax/Parser.swift

```diff
@@ -112,7 +112,7 @@ extension Parser {
         }
 
         let storage: MemoizationStorage
-        if #available(OSX 10.15, *) {
+        if #available(OSX 10.15, iOS 13, *) {
```

Comment:
nit: could we add support for tvOS and watchOS here too?  Pretty sure it's: tvOS 13 and watchOS 6